react/Checkbox/__snapshots__/Checkbox.test.js.snap

```diff
@@ -54,57 +55,11 @@ exports[`Checkbox inputProps should pass through other props to the input 1`] =
   className="root"
 >
   <input
+    checked={false}
     className="input"
     data-automation="first-name-field"
     id="testCheckbox"
-    type="checkbox"
-  />
-  <label
```

Comment:
Trying to figure out why this disappeared from the snapshot 🤔

Reply:
Ohhhh, it's just the weird way the diff is rendering. It's just because I removed a snapshot test.